app/src/test/java/com/stevesoltys/seedvault/metadata/MetadataManagerTest.kt

```diff
@@ -34,6 +35,7 @@ import java.io.IOException
 import kotlin.random.Random
 
 @RunWith(AndroidJUnit4::class)
+@Config(sdk = [29]) // robolectric does not support 30, yet
```

Comment:
Do we know when are they going to?

I think maybe there was a link to some upstream issue - would be nice to have that somewhere, can even be a separate issue here "Update roboelectric once upstream is fixed" or something.

Reply:
No idea. They've just got around to support 29 properly.